### ISSUE TITLE
Decide whether MetaProperty should be included

### ISSUE THREAD

**Issue by RReverser** [RReverser/es-borrowed-props#3]
On one hand, it would be less surprising for user if anything that "quacks like a property" including meta-properties would be expanded like:

``` javascript
{ new.target }
```

into

``` javascript
{ target: new.target }
```

But on other hand those are not real objects.

**Comment by zenparsing** [comment]
I don't think you want to allow those, since a meta property isn't a Reference.

**Comment by RReverser** [comment]
It's not, but from user's perspective it might feel similar and thus cause yet another "gotcha" when it's not supported.
